c2corg_ui/static/js/search/pagination.js

```diff
@@ -47,6 +47,12 @@ app.PaginationController = function($scope, ngeoLocation) {
 
   /**
    * @type {number}
+   * @private
+   */
+  this.maxResultOffset_ = 9970;
```

Comment:
The default limit is 10000, see: https://www.elastic.co/guide/en/elasticsearch/reference/master/search-request-from-size.html